portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -1371,7 +1370,7 @@ $.extend(true, doaj, {
             this.toggleShorten = function(element) {
                 if (!this.component.shortUrl) {
                     var callback = edges.objClosure(this, "updateShortUrl");
-                    this.component.generateShortUrl(callback);
+                    this.component.generateShortUrl(this.component.edge.fullUrl(), callback);
```

Comment:
I've had a look at the downstream change to edges that this introduces, and we can't accept it.  It changes the URL shortening approach, which will have downstream consequences for other users/projects.  I'm not sure why you need to put this `fullUrl` cal here, edges can figure out the full url and ensure that there are no unwanted elements, so we should just leave this as it was.